tb_plugin/torch_tb_profiler/plugin.py

```diff
@@ -206,15 +216,52 @@ def kernel_table_route(self, request):
 
     @wrappers.Request.application
     def trace_route(self, request):
```

Comment:
from the time being, the trace_route include two functionalities: 1), transcode the raw data by adding the gpu utilization content. 2), send the transcode result to the client side. It would be better that if we could make the transcode logic in a separated method or function to make the functionality boundary more clear for maintenance

Reply:
Cool. Thanks. Done.